.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
```

Comment:
We can use values from the matrix to make the job name dynamic.

See here how I have this [on another project](https://github.com/aaemnnosttv/version-assets/blob/9480d7e7666341840e04d831a38223ddf71dc19b/.github/workflows/tests.yml).